smtp/main_test.go

```diff
@@ -209,8 +207,30 @@ func TestDefaultEmailTemplate(t *testing.T) {
 		Params: map[string]string{"buildStatus": "SUCCESS"},
 	}
 
+	// Subject email template
+	subjectTmpl, err := template.New("subject_template").Parse(templateSubject)
+	if err != nil {
+			t.Fatalf("failed to parse subject template: %v", err)
+	}
+
+	subject := new(bytes.Buffer)
+	if err := subjectTmpl.Execute(subject, view); err != nil {
+			t.Fatalf("failed to execute subject template: %v", err)
+	}
+
+	expectedSubject := "Build some-build-id Status: SUCCESS"
```

Comment:
can we keep the test for when no subject template is set and ensure that passes? then create a new test that tests the subject template?

Reply:
Sure, please kindly check https://github.com/GoogleCloudPlatform/cloud-build-notifiers/pull/201/commits/a217c6264186940042c3627c68dc5039608fe8ac

Reply:
When you have a moment, could we merge it now?